### DISCUSSION TITLE
等我长大了，想买一台能看见别人电脑里的文件的电脑

### DISCUSSION THREAD

**Discussion by bsdxm** [zi2ven/riddle-lang#10] [General]
能不能把这个 `// Generated from E:/Riddle-Language/RiddleParser.g4 by ANTLR 4.13.2` 改成 `// Generated from Riddle-Language/RiddleParser.g4 by ANTLR 4.13.2`(等我长大了，想买一台能看见别人电脑里的文件的电脑)，但不要着急，要不然下次改 parser 时又得更新。

**Comment by zi2ven** [comment]
antlr自动生成我有什么办法

**Comment by bsdxm** [comment]
删了

**Comment by zi2ven** [comment]
你为什么不删
